.yamato/project.metafile

```diff
@@ -7,7 +7,6 @@ test_editors:
   - 2021.1
   - 2021.2
   - 2020.3
-  - 2019.4
```

Comment:
I'm not sure whether or not to kick 2019.4 out from Yamato CI/CD since we have plans to potentially downgrade back to 2019.4 in the future